frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx

```diff
@@ -36,5 +36,9 @@ export function useFilteredFacets(wikiInfo: WikiInfoEntry[]) {
       return usefulFacets;
    }, [facets, infoNames]);
 
+   if (title !== null && title === 'Tools') {
+      return facets.slice(7, 10);
```

Comment:
This won't work if we change the order of the facets. We'll want to specifically omit the facets listed on all tool pages. I think this is another place where we can use the new page type field.